syntax = "proto3";


message Asset {

  // A unique string used to calculate the
  // address.
  string rfid = 1;

  // The sizes for each shoe.
  float size_r = 2;
  float size_l = 3;

  // Barcode?
  string sku = 4;
}

// Handle hash collisions.
message AssetContainer {
  repeated Asset entries = 1;
}